Notepad/HomeViewController.swift

```diff
@@ -17,20 +17,32 @@ class HomeViewController: UIViewController {
     }
     
     @IBAction func handleSaveButtonPressed(_ sender: Any) {
-        let title = titleTextField.text
-        print("my title is: \(title!)")
         
-        let notes = bodyTextView.text
-        print("my body is: \(notes!)")
         
-        UserDefaults.standard.set(title, forKey: "Title")
-        UserDefaults.standard.set(notes, forKey: "Body")
+        let alert = UIAlertController(title: "Welcome", message: "Would you like to save your note?", preferredStyle: .alert)
```

Comment:
`"Welcome"` is a weird title to display to the user when he/she presses the save button. The message makes sense though.

Reply:
changed title to Hello which still sounds a bit weird but ig its better

Reply:
In a real app, every time a user presses a button, you will prompt "hello" doesn't make sense. You can just leave it empty